biogrid/libs/biogrid-simulator/src/lib/config/bio-constants.ts

```diff
@@ -8,6 +11,35 @@ export const LARGE_BATTERY = {
   MAX_CAPACITY: 540000,
 };
 
+// Each day a solar panel is roughly charged with electricity
+// for 4hours at a rate of 250 watts ~ 1000watts ~~ 3.6Mj
+// To prolong their life time, solar panels will be expected 
+// to keep a minimum of about 10000j
```

Comment:
what happens if they go below this min?

Reply:
it starts to wear down and degrades the panels